css/style.css

```diff
@@ -0,0 +1,549 @@
+@import url(//db.onlinewebfonts.com/c/a575313c6dc4fd00c1a9506e1c3ea4fc?family=Euclid+Circular+A);
+
+body{
+    font-family: 'Euclid Circular A';

```

Comment:
mistake as in prev homework. 
you should always add an general font family.

in my side font import above not works for some reason (network issue), so I see page with broken serif font instead on sans-serif

Reply:
It is my mistake, I fixed it.